exonum-java-binding/pom.xml

```diff
@@ -519,14 +519,6 @@
                   <message>Java Binding cannot be built on Windows at the moment.
                     You are welcome to hack on ECR-446 and ECR-587!</message>
                 </requireOS>
-                <requireEnvironmentVariable>
-                  <variableName>RUSTFLAGS</variableName>
```

Comment:
Why?

Reply:
I was unable to turn this off within separate profile and @dmitry-timofeev suggested to remove it.

Reply:
But this is temporary for sure. Will be re-written.